docs/mllib-metrics.md

```diff
@@ -0,0 +1,1464 @@
+---
+layout: global
+title: Evaluation Metrics - MLlib
+displayTitle: <a href="mllib-guide.html">MLlib</a> - Evaluation Metrics
+---
+
+* Table of contents
+{:toc}
+
+
+## Algorithm Metrics
+
+Spark's MLlib comes with a number of machine learning algorithms that can be used to learn from and make predictions
+on data. When applying these algorithms, there is a need to evaluate their performance on certain criteria, depending
```

Comment:
Nit, but maybe worth saying we're evaluating the output -- the model -- not the algorithm itself. Evaluations also aren't a property of the application that consumes the model, but of the model.